### PR TITLE
Config Option To Disable Log Files

### PR DESCRIPTION
**Pull request by 1A3Dev**
- Added a config option to disable logging to disk.

### CODE REVIEW

File: src/Config/ConfigManager.cs [hunk 1]
```diff
@@ -121,6 +122,10 @@ private static void CreateConfigElements()
                 "Should UnityEngine.Debug.Log messages be printed to UnityExplorer's log?",
                 false);
 
+            Log_To_Disk = new("Log To Disk",
+                "Should UnityExplorer save log files to the disk?",
+                false);
```
Comment (yukieiji) [inline src/Config/ConfigManager.cs:127]: May I ask that the default value be set to true, as it is undesirable to change the previous behavior?

**Review by yukieiji — CHANGES_REQUESTED** [review]
Thank you for the PR. I just had one point I wanted to check, please confirm.

File: src/Config/ConfigManager.cs (hunk 1 above)
Comment (1A3Dev) [reply src/Config/ConfigManager.cs:127]: Yeah will do, I guessed that would be brought up tbf xD

**Review by yukieiji — APPROVED** [review]

**Comment by yukieiji** [comment]
It can be downloaded from the following URL : https://github.com/yukieiji/UnityExplorer/actions/runs/7564416318

**Pull request merged**